modules/EvseV2G/charger/ISO15118_chargerImpl.cpp

```diff
@@ -47,6 +47,14 @@ void ISO15118_chargerImpl::init() {
         v2g_ctx->tls_security = TLS_SECURITY_PROHIBIT;
         dlog(DLOG_LEVEL_DEBUG, "tls_security prohibit");
     }
+
+    v2g_ctx->tls_key_logging = mod->config.tls_key_logging;
+    v2g_ctx->tls_key_logging_path = mod->config.tls_key_logging_path;
+
+    if (mod->config.tls_key_logging == true) {
+        dlog(DLOG_LEVEL_DEBUG, "tls-key-logging enabled (path: %s)", mod->config.tls_key_logging_path.c_str());
+    }
+    v2g_ctx->certs_path = mod->config.certs_path;
```

Comment:
The certificates from the everest-core/config/certs folder are copied over to the everest-core/build/dist/etc/everest/certs with CMake during installation.

The path info.everest_prefix points to this everest-core/build/dist folder.

```cpp
const std::string EVEREST_ETC_CERTS_PATH = "etc/everest/certs";
v2g_ctx->certs_path = mod->info.everest_prefix + "/" + EVEREST_ETC_CERTS_PATH;
```

Please do not declare the const string in the function, but at the beginning of the file

Reply:
Where is this prefix defined in cmake file? I get "/usr" if I print out the string of mod->info.everest_prefix on charge byte platform.

Reply:
That would result to the file path "/usr/etc/everest/certs" which currently not exists.

Reply:
Here is my suggestion for a quick solution:
```cpp
const std::string EVEREST_ETC_CERTS_PATH = "etc/everest/certs";
[...]
if (mod->info.everest_prefix != "/usr") {
    v2g_ctx->certs_path = mod->info.everest_prefix + "/" + EVEREST_ETC_CERTS_PATH;
} else {
    v2g_ctx->certs_path = "/" + EVEREST_ETC_CERTS_PATH;
}
´´´